addons/website_event_track/controllers/main.py

```diff
@@ -146,11 +148,12 @@ def event_track_proposal_post(self, event, **post):
             'partner_name': post['partner_name'],
             'partner_email': post['email_from'],
             'partner_phone': post['phone'],
-            'partner_biography': escape(post['biography']),
+            'partner_biography': post['biography'],
```

Comment:
Should be sanitized at least. This is a public controller, so it shouldn't be trusted.